.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This should be added to the gitignore as it's specific to VSCode

Reply:
I don't think so atm since this sets the this project's Python runtime in VSCode automatically.